test_truststore.py

```diff
@@ -92,6 +93,8 @@ async def test_sslcontext_api_success_async(host):
 
         assert resp.status == 200
         assert len(await resp.text()) > 0
+    # workaround https://github.com/aio-libs/aiohttp/issues/5426
+    await asyncio.sleep(0.2)
```

Comment:
Shame we have to do this, seems brittle. If it's the only workaround though we must forge on!